src/data/README.md

```diff
@@ -18,7 +18,235 @@ TODO - describe the properties used in DevDot
 
 This file is only used in DevDot. It contains the data and content needed for each product's landing page (`/boundary`, `/consul`, etc.).
 
-TODO - describe the properties used
+### Editing product landing content
+
+Our `<product slug>-landing.json` pages take a "block"-based authoring approach for the main content area. This approach allows authors to write content in JSON instead of JSX. As an overview, each file has three properties, all are required:
+
+```json5
+{
+  "heading": "string", // required
+  "subheading": "string", // required
+  "blocks": [ /*  array of objects, required */ ], 
```

Comment:
Since you note right before this that the properties are required, I think you can leave off the comments about it here. Also found a sneaky extra space character! 😊

```suggestion
  "heading": "string",
  "subheading": "string",
  "blocks": [ /* array of objects */ ], 
```